libopendavinci/src/odcore/data/Container.cpp

```diff
@@ -38,13 +38,15 @@ namespace odcore {
                 m_dataType(UNDEFINEDDATA),
                 m_serializedData(),
                 m_sent(TimeStamp(0, 0)),
-                m_received(TimeStamp(0, 0)) {}
+                m_received(TimeStamp(0, 0)),
+                m_sampleTimeStamp(TimeStamp(0, 0)) {}
 
         Container::Container(const SerializableData &serializableData) :
                 m_dataType(serializableData.getID()),
                 m_serializedData(),
                 m_sent(TimeStamp(0, 0)),
-                m_received(TimeStamp(0, 0)) {
+                m_received(TimeStamp(0, 0)),
+                m_sampleTimeStamp(TimeStamp(0, 0)) {
```

Comment:
Does this mean that when we create a container the sample/creation time is set to the 0th second instead of the actual real-time time?